configs/config.json.example

```diff
@@ -153,6 +153,7 @@
         "type": "EvolvePokemon",
         "config": {
           "enabled": false,
+          "interval": 60,

```

Comment:
Small issue, but I'm not sure if users will be confused by `interval` and `min_evolve_speed`. Perhaps a better name is `log_interval`?